src/azul/terraform.py

```diff
@@ -55,16 +56,16 @@ class TerraformSchema:
 
     @classmethod
     def load(cls, path: Path):
-        with path.open() as f:
-            doc = json.load(f)
+        with gzip.open(path, 'rt') as f:
+            doc = json.loads(f.read())
         return cls(versions=doc['versions'],
                    document=doc['schema'],
                    path=path)
 
     def store(self):
-        with self.path.open('w') as f:
-            json.dump(dict(versions=self.versions,
-                           schema=self.document), f, indent=4)
+        with gzip.open(self.path, 'wt') as f:
+            doc = dict(versions=self.versions, schema=self.document)
+            f.write(json.dumps(doc, indent=4))
```

Comment:
```suggestion
        with gzip.open(self.path, 'wt') as f:
            doc = dict(versions=self.versions, schema=self.document)
            json.dump(doc, f)
```

As above, streaming is more efficient, especially for large files, and there's no point in pretty printing a gzip-ed JSON file.